main.py

```diff
@@ -0,0 +1,267 @@
+"""main.py
+
+objects: screen, player, follower
+
+TODO:
+-walls
+-fix jumping with multiple grounds
+DONE:
+- simple objects for player sprite
+- train constant speed
+- player slows down
+- lose on collision
+- flat level
+"""
+
+import pygame
+import math
+
+# Images
+gameover = pygame.image.load('gameover1.bmp')
+
+# Colors on rgb scale
+BLACK = (0,0,0)
+WHITE = (255,255,255)
+BLUE = (0,0,255)
+DIMGRAY = (105,105,105)
+SLATEGRAY = (112,128,144)
+
+"""Model classes"""
+class Player(object):
+	def __init__(self,x=0,y=0,width=50,height=50,dx=1,dy=.001,shiftdx=0,jumpdy=-.75):
+		# places player centered above the coordinate given
+		self.x = x
+		self.y = y-height
+		#size of player cube
+		self.width = width
+		self.height = height
+		self.dx = dx
+		self.dy = dy
+		#shiftdxs are for moving everything when the player moves
+		self.shiftdx = shiftdx
+		self.jumpdy = jumpdy # variable dy is set to when controller jumps
+
+	def train_wreck(self, train):
+		#train collision function
```

Comment:
Nit-picky and just for future reference, I would also like to see docstrings in the functions also, instead of inline comments. Nice documentation overall, though